tests/bsdtests.c

```diff
@@ -189,6 +189,20 @@ test_long_format(long actual, long expected, const char* format, ...)
 }
 
 void
+_test_sizet(const char* file, long line, const char* desc, size_t actual, size_t expected)
```

Comment:
reads odd but I agree `test_size_t` is not much better, I don't feel strongly either way